lib/overcommit/hook/commit_msg/spellcheck.rb

```diff
@@ -0,0 +1,37 @@
+require 'tempfile'
+
+module Overcommit::Hook::CommitMsg
+  # Checks the commit message for potential misspellings
+  class Spellcheck < Base
+    Misspelling = Struct.new(:word, :suggestions)
+
+    MISSPELLING_REGEX = /^[&#]\s(?<word>\w+)(?:.+?:\s(?<suggestions>.*))?/
+
+    def run
+      # Remove comments from commit message file
+      update_commit_message(commit_message)
```

Comment:
Good call, I've updated the hook accordingly.

As for parallelizing hook runs, any hooks that directly modify the commit message file will probably have to be run serially, which can be specified in the config as discussed in #144.